.evergreen/.evg.yml

```diff
@@ -447,6 +617,24 @@ tasks:
       commands:
         - func: "run plain auth test"
 
+    - name: "aws-auth-test"
+      commands:
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            AUTH: "auth"
+            AWS: "aws"
+            TOPOLOGY: "server"
+        - func: "add aws auth variables to file"
+        - func: "run aws auth smoke test with regular aws credentials"
```

Comment:
As discussed, let's rename the smoke test functions, e.g.

    configure regular aws credentials

Reply:
Done